scripts/tests/lib.sh

```diff
@@ -1,11 +1,23 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
```

Comment:
I am guessing deleting this is a not intentional?

Reply:
oops, seems like an issue with my merge